src/Application/FormMain.cs

```diff
@@ -577,7 +580,7 @@ protected override void OnLayout(LayoutEventArgs levent)
             //this.taskList.Location = new Point(0, top + this.xmlTreeView1.Height + this.resizer.Height);
             this.tabControlLists.Size = new Size(w, this.tabControlLists.Height);
             this.tabControlLists.Location = new Point(0, top + this.tabControlViews.Height + this.resizer.Height);
-            base.OnLayout(levent);
+            //base.OnLayout(levent);
```

Comment:
why is base.OnLayout removed?

Reply:
cause this was useless.
you already did it on the first line of current method.
one of them is duplicated and affects the performance

Reply:
ah, got it.  Please delete the line then, thanks.